Dockerfile.new

```diff
@@ -0,0 +1,221 @@
+# Note: once we have fully transitioned to the Go server, this will replace the old Dockerfile
```

Comment:
It might be misleading to have this in the root directory. Should we move it to the `llama`, `scripts` directory or somewhere else for now?